files/build_templates/init_cfg.json.j2

```diff
@@ -32,7 +32,7 @@
             "tsa_enabled": "false"
         }
     },
-{%- set features = [("bgp", "enabled", false, "enabled"),
+{%- set features = [("bgp", "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}disabled{% else %}enabled{% endif %}", false, "enabled"),
```

Comment:
In order to allow the data value to be generated with variable in the init_cfg.json after build,  need to have {% raw  %}  .... {% endraw %}.  Such as below
{% raw %} "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}disabled{% else %}enabled{% endif %}", false, "enabled")",{% endraw %}